test/basic-querying.test.js

```diff
@@ -57,8 +60,10 @@ describe('basic-querying', function() {
       User.destroyAll(done);
     });
 
+    var id = isCassandraConnector ? uuidV4() : 1;
```

Comment:
What about defining `id` explicitly as a number to avoid connector-specific default id types?